pkg/test/echo/server/forwarder/grpc.go

```diff
@@ -131,6 +133,11 @@ func (c *grpcCall) makeRequest(ctx context.Context, cfg *Config, requestID int)
 			echo.WriteBodyLine(&outBuffer, requestID, line)
 		}
 	}
+	for k, v := range header {
+		for _, vv := range v {
+			echo.WriteBodyLine(&outBuffer, requestID, fmt.Sprintf("ResponseHeader=%s:%s", k, vv))
```

Comment:
I think this will show up as under `body`. Seems like we do `echo.ResponseHeaderField.WriteKeyValueForRequest(outBuffer, requestID, key, value)` for http which may be more appropriate.

We should probably switch to json eventually..

Reply:
Fixed, I realized it didn't look right when testing...

Regarding json - I agree, even to proto since this is a grpc method... With some way to opt-in so we don't change all the tests. 

One thing to keep in mind ( not sure people are aware ) - there is a fork of this implemented in C++ in grpc,
which they use for testing with Istio. We need to coordinate with them if we make backward incompatible
changes or add features.